pkg/webhooks/sgx/sgx.go

```diff
@@ -35,6 +35,7 @@ var ErrObjectType = errors.New("invalid runtime object type")
 type Mutator struct{}
 
 const (
+	epcLimitKey              = "epc-limit.nri.io"
```

Comment:
to get a shorter line
```suggestion
	epcLimitKey              = "epc-limit.nri.io/container"
```